multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -639,7 +613,32 @@ def cocoeval_torchmetrics(outputs):
     return map_metric.compute()
 
 
-def cocoeval_pycocotools(outputs, data, anno_file, cache_path, metrics):
+def cocoeval_pycocotools(
+    outputs: List, data: pd.DataFrame, anno_file: str, cache_path: str, metrics: Optional[Union[str, List]]
+):
+    """
+    Evaluate predictor's output using pycocotool's mAP implementation.
```

Comment:
It might be good to add a website link to cocoapi here, https://github.com/cocodataset/cocoapi